readme.md

```diff
@@ -110,7 +110,7 @@ console.log(getProperty(object, escapedPath));
 
 ### deepKeys(object)
 
-Returns an array of every path. Plain objects are deeply recursed and are not themselves included.
+Returns an array of every path. Plain not empty objects are deeply recursed and are not themselves included. Empty objects or objects without values are included, e.g [], {}, null, undefined or NaN.
```

Comment:
This is more confusing than it needs to be. Just say that non-empty plain objects and arrays are not themselves included.